mobify/hora.js
# Add View Desktop click event tracking

Status: **Ready for Review**
Owner: @marlowpayne
Reviewers: @ericmuyser @scalvert @Helen-Mobify 

## Changes
- Carnival has asked us to track clicks on their "View Desktop" link. I have added this as a general Hora event for tracking.

## Jira Tickets:
- https://mobify.atlassian.net/browse/CCL-435

## Todos:

### Feedback:
_none so far_

## How to Test
- Checkout this branch
- Run `grunt test` and/or `grunt test:browser`
- Make sure the code makes sense
